stdlib/etc/0/etc.rbs

```diff
@@ -0,0 +1,745 @@
+# <!-- rdoc-file=ext/etc/etc.c -->
+# The Etc module provides access to information typically stored in files in the
+# /etc directory on Unix systems.
+#
+# The information accessible consists of the information found in the
+# /etc/passwd and /etc/group files, plus information about the system's
+# temporary directory (/tmp) and configuration directory (/etc).
+#
+# The Etc module provides a more reliable way to access information about the
+# logged in user than environment variables such as +$USER+.
+#
+# ## Example:
+#
+#     require 'etc'
+#
+#     login = Etc.getlogin
+#     info = Etc.getpwnam(login)
+#     username = info.gecos.split(/,/).first
+#     puts "Hello #{username}, I see your login name is #{login}"
+#
+# Note that the methods provided by this module are not always secure. It should
+# be used for informational purposes, and not for security.
+#
+# All operations defined in this module are class methods, so that you can
+# include the Etc module into your class.
+#
+module Etc
+  # <!--
+  #   rdoc-file=ext/etc/etc.c
+  #   - confstr(p1)
+  # -->
+  # Returns system configuration variable using confstr().
+  #
+  # *name* should be a constant under `Etc` which begins with `CS_`.
+  #
+  # The return value is a string or nil. nil means no configuration-defined value.
+  #  (confstr() returns 0 but errno is not set.)
+  #
+  #     Etc.confstr(Etc::CS_PATH) #=> "/bin:/usr/bin"
+  #
+  #     # GNU/Linux
+  #     Etc.confstr(Etc::CS_GNU_LIBC_VERSION) #=> "glibc 2.18"
+  #     Etc.confstr(Etc::CS_GNU_LIBPTHREAD_VERSION) #=> "NPTL 2.18"
+  #
+  def self.confstr: (::Integer p1) -> ::String?
```

Comment:
Could you make all of the methods module function (`self?`)?

Reply:
Thanks. Fixed via 186b2947.

Are test cases for module functions necessary? If yes, can you tell me how to test module functions?